numpy/core/_type_aliases.py

```diff
@@ -224,9 +222,7 @@ def _set_array_types():
 
 # Add additional strings to the sctypeDict
 _toadd = ['int', ('float', 'double'), ('complex', 'cdouble'), 
-          'bool', 'object',
-          'str', 'bytes', ('a', 'bytes_'),
-          ('int0', 'intp'), ('uint0', 'uintp')]
+          'bool', 'object', 'str', 'bytes', ('a', 'bytes_')]
```

Comment:
`np.dtype('a')` also seems ripe for removal, never seen it before and looks very odd. Does anyone know where that came from?

Reply:
I've actually noticed `np.dtype('a')` but I couldn't figure out its origin and purpose 😄 - therefore I decided to keep it.

Reply:
Looks like this goes back to https://github.com/numpy/numpy/commit/9ac3b0e5c2e181873221ff0160b882772ea9d435. I have no idea why Travis added it and I don't know whether issues from the old trac issue tracker are still visible somewhere. Github search only finds 12 code usages of `"np.dtype('a')"`. I say remove it.

Reply:
Presumably it is meant to mean "ascii"... if almost nobody uses, it shouldn't matter, unlik `"c"` it dosn't eve have a special meaning.